modules/tripleliftBidAdapter.md

```diff
@@ -3,7 +3,7 @@
 ```
 Module Name:  Triplelift Bid Adapter
 Module Type:  Bidder Adapter
-Maintainer: bzellman@triplelift.com
+Maintainer: csmith+s2s@triplelift.com
```

Comment:
For the record, we would prefer that maintainer addresses be groups rather than individuals.